packages/components/pie-list/src/pie-list-item/list-item.scss

```
@@ -1 +1,67 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+
+:host {
+  // These variables can be overridden by the parent list component
+  --list-item-leading-margin: var(--dt-spacing-c);
+  --list-item-trailing-margin: var(--dt-spacing-c);
+  --list-item-hover-bg: var(--dt-color-hover-1);
+  --list-item-active-bg: var(--dt-color-active-1);
```

doesn't seem used yet but the tokens should be 

```suggestion
  --list-item-hover-bg: var(--dt-color-hover-01);
  --list-item-active-bg: var(--dt-color-active-01);
```